FlowCrypt/Controllers/Bootstrap/InvalidStorageViewController.swift

```diff
@@ -85,3 +53,73 @@ final class InvalidStorageViewController: UIViewController {
         }
     }
 }
+
+extension InvalidStorageViewController: ASTableDelegate, ASTableDataSource {
+    func tableNode(_: ASTableNode, numberOfRowsInSection _: Int) -> Int {
+        Parts.allCases.count
+    }
+
+    func tableNode(_ node: ASTableNode, nodeBlockForRowAt indexPath: IndexPath) -> ASCellNodeBlock {
+        return { [weak self] in
+            guard let self = self, let part = Parts(rawValue: indexPath.row) else {
+                return ASCellNode()
+            }
+
+            let insets = UIEdgeInsets(top: 8, left: 16, bottom: 8, right: 16)
+            switch part {
+            case .screenTitle:
+                return SetupTitleNode(
+                    SetupTitleNode.Input(
+                        title: "invalid_storage_title".localized
+                            .attributed(
+                                .bold(18),
+                                color: .mainTextColor,
+                                alignment: .center
+                            ),
+                        insets: insets,
+                        backgroundColor: .backgroundColor
+                    )
+                )
+            case .title:
+                return SetupTitleNode(
+                    SetupTitleNode.Input(
+                        title: "invalid_storage_text".localized
+                            .attributed(
+                                .regular(16),
+                                color: .mainTextColor,
+                                alignment: .center
+                            ),
+                        insets: insets,
+                        backgroundColor: .backgroundColor
+                    )
+                )
+            case .description:
+                return SetupTitleNode(
```

Comment:
Looks like if we have very long stack trace text here, we will have very long node (big height) and button will be hidden from user. User have to scroll down to see button. In my original version button is always on screen, independent of stack trace node height.

Reply:
Can keep this behaviour, but should we? 
Apple Human Interface Guidelines recommends to use table views for scrollable layout. Also it's more common for iOS users to scroll if content is higher than screen size. And it's quite rare to have such long localized description

Reply:
The exception that caused us to put this in place was indeed quite long, causing a long scroll. 

I guess it's fine if the button ends up on the bottom, where user has to scroll. It's more user friendly when it's on top, but it's not too bad on a screen that should be rarely, if ever, seen.